tools/accuracy_checker/accuracy_checker/launcher/tf_launcher.py

```diff
@@ -48,20 +46,22 @@ def __init__(self, config_entry, *args, **kwargs):
         try:
             import tensorflow # pylint: disable=C0415
             from tensorflow.python.saved_model import tag_constants # pylint: disable=C0415
-            if tensorflow.__version__ >= '2.0.0':
-                self.tf = tensorflow.compat.v1
-                self.tf_gfile = tensorflow.io.gfile
-            else:
-                self.tf = tensorflow
-                self.tf_gfile = tensorflow.gfile
+            self.tf = tensorflow.compat.v1 if tensorflow.__version__ >= '2.0.0' else tensorflow
```

Comment:
why do you need these changes? and why do you remove gfile?

Reply:
fixed